sharp/CmdInstallEverest.MiniInstaller.cs

```diff
@@ -164,6 +173,38 @@ class MiniInstallerProxy : MarshalByRefObject {
 
                 }
             }
+
+            private void BootNative(MiniInstallerBridge bridge) {
+                string installerPath = Path.Combine(bridge.Root,
+                    PlatformHelper.Is(Platform.Windows) ? "MiniInstaller-win.exe" :
+                    PlatformHelper.Is(Platform.Linux)   ? "MiniInstaller-linux" :
+                    PlatformHelper.Is(Platform.MacOS)   ? "MiniInstaller-osx" :
+                    throw new Exception("Unknown OS platform")
+                );
```

Comment:
Is the current plan to dump all the installer files into the Celeste directory? I'm mildly concerned about things getting messy that way, maybe a separate subdirectory might be a good idea... unless those are fully self-contained single file builds which don't clutter up the Celeste folder 😅

Reply:
These are only the apphosts, so other than these three files + `MiniInstaller.dll` the builds won't differ.

Note though that "Install from ZIP" is currently broken (at least on Linux), so I didn't get a chance yet to see if this bridge code actually works in the wild (I'm gonna run a test on Windows shortly though).